akvo/templates/myrsr/project_editor/section_5.html

```diff
@@ -32,6 +32,11 @@
                             </div>
                         </div>
                     </div>
+                    {% include boolean_input with obj=project field='aggregate_to_parent' %}
+                {% endif %}
+                {% if project.children.count > 0 %}
```

Comment:
Also, it only shows published parents / children (this is the result of my changes in the hierarchy issue). Maybe we should make new functions for all children / parents and published children parents. Because in the project editor, you would like to show unpublished projects as well.